### PR TITLE
Get nosetests working after removing the setup.py change

### PR DESCRIPTION
**Pull request by sb2nov**
This is a temporary change as we work towards getting maven to work across languages and building and testing both the python and java code. 

R: @aaltay PTAL

CC: @dhalperi

Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by sb2nov** [comment]
Created https://issues.apache.org/jira/browse/BEAM-1449 for the long term strategy.

**Comment by davorbonaci** [comment]
Is it possible to add a presubmit check of type `diff -q <first copy> <second copy>` somewhere?

LGTM otherwise.

**Comment by aaltay** [comment]
LGTM. I suppose we can update `pom.xml` to include the `diff` test suggested by @davorbonaci. Could we do that in a separate PR, since we are planning to address this in a better soon?

**Comment by sb2nov** [comment]
+1 to creating a different PR for the diff test, but open to otherwise.

**Comment by aaltay** [comment]
Should we also move `sdks/python/apache_beam/coders/standard_coders.yaml` to `tests/data`?

**Comment by sb2nov** [comment]
@aaltay PTAL

**Comment by aaltay** [comment]
LGTM. I will merge it as a short term solution.

File: sdks/python/apache_beam/coders/standard_coders_test.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def quote(s):
 
 
 STANDARD_CODERS_YAML = os.path.join(
-    os.path.dirname(__file__), 'standard_coders.yaml')
+    os.path.dirname(__file__), '..', 'tests/data/standard_coders.yaml')
```
Comment (dhalperi) [inline sdks/python/apache_beam/coders/standard_coders_test.py]: I think you need to os.path.join all the path components, including "tests" and "data" separately.

Comment (sb2nov) [inline sdks/python/apache_beam/coders/standard_coders_test.py]: Thanks, #1972 should address it.

**Pull request closed without merging**